src/file.js

```diff
@@ -0,0 +1,30 @@
+const fs = require("fs");
+
+function readFile(filepath) {
+    try {
+        return fs.readFileSync(filepath, 'utf8');
+    }  catch (e) {
+        console.log(e);
+        throw e
+    }
+}
+
+function writeFile(filePath, fileName, fileContent) {
+    try {
+        if (!fs.existsSync(filePath)) {
+            console.warn("Output dir doesnt exists. Creating it...");
```

Comment:
I Belive we can use the new log module here so we can centralize the execution of console.log and console.warn